aspnetcore/fundamentals/servers/index.md

```diff
@@ -102,10 +106,51 @@ When using [Visual Studio](https://www.visualstudio.com/vs/), [Visual Studio for
 
 When launching an app from a command prompt in the project's folder, [dotnet run](/dotnet/core/tools/dotnet-run) launches the app and server (Kestrel and HTTP.sys only). The configuration is specified by the `-c|--configuration` option, which is set to either `Debug` (default) or `Release`. If launch profiles are present in a *launchSettings.json* file, use the `--launch-profile <NAME>` option to set the launch profile (for example, `Development` or `Production`). For more information, see the [dotnet run](/dotnet/core/tools/dotnet-run) and [.NET Core distribution packaging](/dotnet/core/build/distribution-packaging) topics.
 
+## HTTP/2 support
+
+[HTTP/2](https://httpwg.org/specs/rfc7540.html) is supported with ASP.NET Core in the following deployment scenarios:
+
+::: moniker range=">= aspnetcore-2.2"
+
+* [Kestrel](xref:fundamentals/servers/kestrel#http2-support)
+  * Operating system
+    * Windows Server 2012 R2/Windows 8.1 or later
+    * Linux with OpenSSL 1.0.2 or later (for example, Ubuntu 16.04 or later)
+    * HTTP/2 will be supported on macOS in a future release.
+  * Target framework: ASP.NET Core 2.2 or later
```

Comment:
.NET Core 2.2, not ASP.NET Core 2.2. .NET 4.x doesn't support ALPN or HTTP/2.

Reply:
Should it say "Shared framework: ..." since we don't want to confuse TFMs with this remark?

Reply:
TFM is more applicable. In-proc and ALPN both have dependencies in .NET Core 2.2 and don't support .NET 4.x.